cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -592,6 +592,12 @@ class QueueNetworkingSchema(BaseNetworkingSchema):
 class SlurmQueueNetworkingSchema(QueueNetworkingSchema):
```

Comment:
let's leave the changes to subnet_ids out of this PR for now

Reply:
Ok, changed back